docs/en/connector-v2/sink/Jdbc.md

```diff
@@ -197,6 +198,10 @@ When data_save_mode selects CUSTOM_PROCESSING, you should fill in the CUSTOM_SQL
 
 Enable upsert by primary_keys exist, If the task has no key duplicate data, setting this parameter to `false` can speed up data import
 
+### use_copy_statement [boolean]
+
+Use `COPY ${table} FROM STDIN` statement to import data. Only drivers with getCopyAPI () interface connections are supported.
```

Comment:
Could you list which driver supported this interface. I think user maybe not clear this one.

Reply:
fix docs:
Use `COPY ${table} FROM STDIN` statement to import data. Only drivers with `getCopyAPI()` method connections are supported. e.g.: Postgresql driver `org.postgresql.Driver`.